codec/src/test/resources/io/netty/handler/codec/xml/sample-04.xml

```diff
@@ -649,20 +649,19 @@
         </plugin>
         <plugin>
           <artifactId>maven-dependency-plugin</artifactId>
-          <version>2.8</version>
+          <version>2.10</version>
         </plugin>
         <plugin>
           <artifactId>maven-assembly-plugin</artifactId>
-          <version>2.4</version>
+          <version>2.6</version>
         </plugin>
         <plugin>
-          <!-- Do NOT upgrade -->
           <artifactId>maven-jxr-plugin</artifactId>
-          <version>2.2</version>
+          <version>2.5</version>
```

Comment:
Yes, you are right, it only runs it when generating the site now, rolling back this one.